deliverables/README.md

```diff
@@ -9,60 +9,123 @@ To answer questions, please just put the answer on the next line after the quest
 Example:  
 
 **What is your name?**
-Sir Student of Ruby
+Nolawee Mengist
 
 **What is your quest?**
-I seek the Rails  
+Make a stupid amount of money
 
 **What is your favorite color?**
-~~Green~~ edit:  No, blue.  Blue!  
+Red 
 
 For projects on Code Academy, or other external exercises, please just paste a link into your deliverables file where we can see the result.  
 
 ### Section 1: The Front End
 #### Required - Answer these questions in the deliverables file, and complete the following. 
-What is the role of HTML in a web page?  
+What is the role of HTML in a web page? 
+It is the standard markup language used for making webpages. HTML is the backbone of all websites and translates images, video and text into a format that your browser can understand. 
+
 What is the role of CSS in a web page?  
-What is the role of Javascript in a web page?  
+CSS stands for Cascading Style Sheets and is used to adjust the visual asthetics of a webpage.
 
+What is the role of Javascript in a web page?  
+Javascript is used to affect the behavior of HTML pages. It allows you to let the designer dynamically add functionality to their website.
 ---
 
 ### Section 2: HTML and CSS 101
 
 #### meta
 Why do we separate HTML and CSS?  
-
+We seperate HTML and CSS to make maintaining code easier, keeps code more efficent and it's standard programming practice.
 #### HTML
-What are classes and IDs (and how are they different)?  
+What are classes and IDs (and how are they different)?
+They both indicate differt parts of your webpage that can be modified using CSS or Javascript. When one manipulates a class using CSS or Javascript they are modifying all the Ids within that class. When one edits a ID using CSS or Javascript they only manipulate one specific element of a webpage. 
+
 What are elements?  
+HTML elements are individual components of a webpage. They tend to mark the begining and end of a webpage. 
+
 What are tags?  
+HTML tags apply meaning to elements of a webpage and help apply context to the actual code within the webpage. 
+
 What are attributes?  
+Extra pieces of information that is a modifier of an HTML element. 
+
 What are forms?  
+They provide a way for websites to capture information from users and to process requests
+
 What is a div?  
+Defines a division or a section in an HTML document and is used to group block-elements to format them with CSS
 
 What are selectors?  
+Selectors in CSS refer to element(s) you want to style.
```

Comment:
Once again, we can go super-technical with this.  A "selector" is a string of characters that the browser compares against elements on the page to select CSS rules to apply to them.  

Some javascript libraries use selectors as well to target sets of elements.  For example, jQuery(".foo > .bar") selects all elements of class "bar" directly inside an element of class "foo".  Shortcut for `jQuery` function is `$`, so you'll see that more often like `$(".foo > .bar")`